### ISSUE TITLE
Tutorial 9 fails with Clojure 1.8

### ISSUE THREAD

**Issue by samestep** [magomimmo/modern-cljs#373]
Clojure 1.8 [fixed a bug](http://dev.clojure.org/jira/browse/CLJ-1456) causing extraneous arguments in `throw` forms to be ignored, exposing a [bug in `shoreleave-remote-ring`](https://github.com/magomimmo/shoreleave-remote-ring/blob/1a1c473283319e01c177f96aaa61d3e6535773fe/src/shoreleave/middleware/rpc.clj#L46-L47) and causing said library to fail when compiled in Clojure 1.8. This makes Tutorial 9 impossible to finish with Clojure 1.8, because an exception is thrown [when the IFDE is launched](https://github.com/magomimmo/modern-cljs/blob/master/doc/second-edition/tutorial-09.md#launch-ifde). [Tim Goodwin's pull request](https://github.com/magomimmo/shoreleave-remote-ring/pull/1) fixes the issue. Could you please merge that pull request and publish an updated version of `shoreleave-remote-ring` to Clojars?

**Comment by magomimmo** [comment]
thanks for the notice. It's very unusual that I did not receive the notification about the pull request regarding shoreleave. I'll have some time this evening or tomorrow to get into it. thanks again...

**Comment by samestep** [comment]
Do you think you might have time to fix this issue soon?

**Comment by magomimmo** [comment]
@samestep hopefully in the week end...sorry for the delay....

**Comment by magomimmo** [comment]
Hi @samestep, I just updated shoreleave-remote-ring to 0.3.2 which now includes the fix by @trgoodwin 
I did not test all the tutorials with clj 1.8.0 and cljs 1.8.51.  I soon as I'll have more time I'll update all the dependencies. At the moment, if you want to use clj 1.8.0 and cljs 1.8.51 you just need to substitute 0.3.2 to 0.3.1 release of shoreleave-remote-ring lib and everything works (at least in this tutorial).
